docs/integrations/cloud/aws-amplify.mdx

```diff
@@ -48,13 +50,13 @@ This approach enables the synchronization of secrets from Infisical to AWS Ampli
   </Step>
 </Steps>
 
-## Sync Secrets Using AWS SSM Parameter Store
+## Sync Secrets Using AWS SSM Parameter Store 
 
-[Environment secrets](https://docs.aws.amazon.com/amplify/latest/userguide/environment-variables.html#environment-secrets) are similar to environment variables, but they are AWS Systems Manager (SSM) Parameter Store key-value pairs that can be encrypted. By using Infisical AWS SSM integration, you can synchronize secrets from Infisical to SSM, which in turn synchronizes secrets to your AWS Amplify environment secrets.
+Another approach to use secrets from Infisical in AWS Amplify is to utilize AWS Parameter Store. At high level, you begin by using Infisical's AWS SSM integration to sync secrets from Infisical to AWS SSM. You then instruct AWS Amplify to consume those secrets from AWS SSM as environment variables.
```

Comment:
@maidul98 you need to explicit say these are as environment secrets that's why i added the link to aws. They can't access it normally